sdk/src/wallet/account/operations/transaction/prepare_output.rs

```diff
@@ -389,8 +389,10 @@ impl Account {
 }
 
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct OutputParams {
     pub recipient_address: Bech32Address,
+    #[serde(with = "crate::utils::serde::string")]
     pub amount: u64,
```

Comment:
Are the default tags not needed?

Reply:
They probably are, completely forgot them, thanks

Reply:
Done